app/src/main/java/com/eventyay/organizer/utils/LinkUtils.java

```diff
@@ -0,0 +1,12 @@
+package com.eventyay.organizer.utils;
+
+public class LinkUtils {
+
+    private LinkUtils() {
+        // Never Called
+    }
+
+    public static String getToken(String url) {
```

Comment:
What will you do when you have to handle link to verify password?

Create an enum Destination with identifier and return it

```kotlin
enum Destination {
  PASSWORD_VERIFY,
  PASSWORD_RESET
}

class DestinationWithIdentifier(destination, identifier: String)

DestinationWithIdentifier getDestination(String url) {
  if (url is password reset)
    return DestinationWithIdentifier(PASSWORD_RESET, token)
  else if (url is password verify)
    return DestinationWithIdentifier(PASSWORD_VERIFY, token)
  else
    return null
}
```

Reply:
Updated.